Gemfile

```diff
@@ -72,6 +72,7 @@ gem 'aws-sdk-core', '~> 2.2.15'
 # Optional Services.
 gem 'omniauth-37signals'          # BasecampAgent
 gem 'omniauth-wunderlist', github: 'wunderlist/omniauth-wunderlist', ref: 'd0910d0396107b9302aa1bc50e74bb140990ccb8'
+gem 'omniauth-lifx'
```

Comment:
I wonder if we should start defaulting new dependencies off?  If only 1% of our users need this service, it's a shame to add a new Gem for it.  Thoughts?

The Agent can explain that dependencies are needed-- see ` #{twitter_dependencies_missing if dependencies_missing?}` in the Twitter agents.